### ISSUE TITLE
Slowness after update 

### ISSUE THREAD

**Issue by taman0753** [warpdotdev/Warp#1666]
### Describe the bug

There is slowness in the new version. It gets hang.

### To Reproduce

There is slowness in the new version. It gets hang.

### OS Version

12.4

### Shell Version

zsh 5.8.1

### Warp Version

v0.2022.07.25.09.05.stable_00

### Does this block you from using Warp daily?

Yes

**Comment by sathwikmatsa** [comment]
I face this issue post update today. It hangs frequently and unusable.

**Comment by francescob** [comment]
same for me on macOS 12.4, using fish as a shell

**Comment by M1chlCZ** [comment]
Same thing for me with ZSH shell. 
macOS 12.5, M1 OG

**Comment by alokedesai** [comment]
Hey all, thanks for reporting and apologies for the issues here. We're actively looking into this and should have updates shortly. Could you clarify where the hangs happen? Did you download Warp via the website (www.warp.dev) or via homebrew?

**Comment by M1chlCZ** [comment]
For me it was installation from website and it hangs while scrolling and of course when you are calling nano and htop. Which is ongoing issue. Nothing like that is happening in Terminal app.

**Comment by alokedesai** [comment]
@M1chlCZ Thanks for the quick response--Can you confirm that you are using the arm version of Warp? To check, you can go to Activity Monitor, and check whether the "Kind" column says "Apple". 

Also, just to sanity check, this only started happening with the most recent release?

![image](https://user-images.githubusercontent.com/4110292/181559007-47dd9d6c-10c9-4832-bf54-34977b6d7258.png)

**Comment by alokedesai** [comment]
@atouchsimo that's a really interesting find that it seems only related to ssh. @M1chlCZ, are you also only able to repro when sshing?

**Comment by M1chlCZ** [comment]
@alokedesai Yes I am using M1 Warp
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/66187879/181565160-7d3a5b59-0f59-4de2-bfe7-83b0ad4eb49c.png">

And yes it's happening while SSHing with the most recent release

**Comment by taman0753** [comment]
@alokedesai  Same issue is happening with me as well. Scrolling is very slow , also when I ssh into some machine it hangs a lot. yes its the issue with the latest release (v0.2022.07.25.09.05.stable_00)

**Comment by alokedesai** [comment]
@atouchsimo @M1chlCZ @taman0753 ooc what shell is your remote machine using?

**Comment by taman0753** [comment]
zsh 5.8.1 @alokedesai

**Comment by M1chlCZ** [comment]
@alokedesai For me it's Bash

**Comment by alokedesai** [comment]
Thanks for all your help everyone! We're going to start by rolling back Warp to last week's release to unblock y'all. We still need to find the culprit though, of course. We'll be following up shortly to try to investigate more about what's causing this.

In the meantime, @atouchsimo (or anyone else who sees this) could I send you a beta version of Warp with the rollback to sanity check everything is working before we rollback the stable channel more broadly?

**Comment by taman0753** [comment]
Yes you can

**Comment by taman0753** [comment]
@alokedesai Added my email to GB.

**Comment by alokedesai** [comment]
@taman0753 Thanks! Just sent to you

**Comment by alokedesai** [comment]
@atouchsimo Thank you! We're rolling this out to stable right now. You should a new update appear within the next 10 minutes.

**Comment by taman0753** [comment]
@alokedesai  Did a basic sanity , working as expected.

**Comment by alokedesai** [comment]
Thank you everyone for the help and for flagging this issue! We still need to identify the culprit and may be in touch for more help trying to repro on our end so we can fix and rollforward :D

**Comment by alokedesai** [comment]
@taman0753 It should be available in the current stable version you are using. If go to settings and hit "check for update", Warp should show a button to update once it's fetched the new build

**Comment by szgupta** [comment]
Hey @taman0753, I sent you an email to debug further :)

**Comment by szgupta** [comment]
Hey everyone, you should now be able to downgrade to the `v0.2022.07.18.09.06.stable_01` version. Thank you for your patience and sorry about any inconvenience this has caused. We'll update this issue once we've identified the problem.

**Comment by szgupta** [comment]
Hey folks, you should now be able to update to `v0.2022.07.25.09.05.stable_01`. We're sorry for the inconvenience this has caused. I'll close this issue for now, but feel free to re-open if you are still experiencing the same problem (or any other problem with the latest release).

**Comment by runforprogram** [comment]
has the problem  on  * v0.2022.07.25.09.05.stable_01 * 
 with ZSH shell macOS 12.5 M1 Pro . when chose theme it is hang.

**Comment by szgupta** [comment]
Hey @runforprogram, do you mind opening a new issue for this with the following:
- repro steps (and if possible, a video of the issue)
- does the app hang (loading beachball cursor) or does the shell hang?

Let's diagnose in that issue!

**Comment by runforprogram** [comment]
> 

the new issue . [https://github.com/warpdotdev/Warp/issues/1669]

**Comment by szgupta** [comment]
Hey @taman0753 and @atouchsimo, I emailed you both a version of Warp where we think we fixed the issue (while still supporting the ability to open file links). Please let me know what you think :)
